CMakeLists.txt

```diff
@@ -152,12 +152,13 @@ endif()
 option(SDLMIXER_MP3 "Enable MP3 music" ON)
 
 cmake_dependent_option(SDLMIXER_MP3_MINIMP3 "Support loading MP3 music via minimp3" ON SDLMIXER_MP3 OFF)
+cmake_dependent_option(SDLMIXER_MP3_DRMP3 "Support loading MP3 music via dr_mp3" OFF SDLMIXER_MP3 OFF)
```

Comment:
Are these mutually exclusive?

Reply:
I guess. However, the same question can be asked for mpg123 and minimp3.